Worktrolly/worktrolly
# feat(Filter):Added Filter Component with Query

Signed-off-by: sandeep-v1404 <valpasanisandeep@gmail.com>

### Functionality:
Filter Task by Combination of Assignee, Category, Priority. Status , Difficulty

### Solution:

- Implement query to get data from firebase Firestore DB . 
- Add Sidebar at left with checkboxes to filter with options . 
- Filter Button to filter Data . 

### Risk level:

### How to test:

1. ng serve
2. Open a Category in Dashboard and Click Filter Icon in Navigation bar of Tasks . 
3. Add filter Options and Click Filter . 
4. Check Responsiveness using Chrome Dev Tools

### Images:

- Filter with Options

![filterWork](https://user-images.githubusercontent.com/57038543/103439329-f328ad00-4c61-11eb-909e-121b2b3ba7e7.png)

- Select Multiple Filters

![filterWork1](https://user-images.githubusercontent.com/57038543/103439336-050a5000-4c62-11eb-9b88-67d3e9ca28e6.png)

- Responsiveness

![filterResponsive](https://user-images.githubusercontent.com/57038543/103439343-16535c80-4c62-11eb-86e2-c1ad9e60d2c7.png)
